packages/beacon-state-transition/src/allForks/util/epochContext.ts

```diff
@@ -439,19 +439,17 @@ export class EpochContext {
 
     const epochStartSlot = computeStartSlotAtEpoch(epoch);
     for (let slot = epochStartSlot; slot < epochStartSlot + SLOTS_PER_EPOCH; slot++) {
```

Comment:
at api side we call this function in a for loop with a list of `validatorIndices`. I wonder if we can do that in batch, i.e supporting a list of `ValidatorIndex` in this function too.

Reply:
Good idea. Lets merge this first and attempt that separately.